app/src/main/kotlin/com/wire/android/ui/home/HomeTopBar.kt

```diff
@@ -0,0 +1,91 @@
+package com.wire.android.ui.home
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Menu
+import androidx.compose.material3.DrawerState
+import androidx.compose.material3.DrawerValue
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.rememberDrawerState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.wire.android.R
+import com.wire.android.ui.common.SearchBarUI
+import com.wire.android.ui.common.UserProfileAvatar
+import com.wire.android.ui.theme.title01
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@ExperimentalMaterial3Api
+@Composable
+fun HomeTopBar(
+    @StringRes title: Int?,
+    isSearchable: Boolean,
+    drawerState: DrawerState,
+    viewModel: HomeViewModel
+) {
+    val scope = rememberCoroutineScope()
+    val titleText = stringResource(id = title ?: R.string.conversations_screen_title)
+
+    Column(
+        Modifier
+            .fillMaxWidth()
+            .background(MaterialTheme.colorScheme.background)
+    ) {
+        Row(
```

Comment:
So this is now fully custom? Why don't use a ready-made `CenterAlignedTopAppBar`?

Reply:
problem that there should be Search field at the bottom. So i tried to have `Column(CenterAlignedTopAppBar, SearchField)` but it looks a bit different from the designs. Basically because `CenterAlignedTopAppBar` has its own height and bottom padding, so the distance between top part of the TopBar and Search field is bigger than should.

Reply:
I can't build the app on my side to check how it would look like. Could you maybe attach a screenshot of it?